app/models/call_off_contract.rb

```diff
@@ -5,6 +5,6 @@ class CallOffContract < ApplicationRecord
   belongs_to :lead_provider
 
   def band_a
-    participant_bands.order(:min).first
+    participant_bands.order("min ASC NULLS FIRST").first
```

Comment:
We could also use scope on the ```ParticipantBand```  

i.e. ```scope :nulls_first, -> { order("min asc nulls first") }```

and then do the following:

```
def band_a
  participant_bands.nulls_first.first
end
```

so in future could be reused for band_b, band_c etc

```
def band_b
  participant_bands.nulls_first.second
end
```

Reply:
Yep. Scope is a good idea. I've modified to something similar to your suggestion.